XRTK.WebXR/Packages/com.xrtk.webxr/package.json

```diff
@@ -10,18 +10,17 @@
     "mixed",
     "reality"
   ],
-  "version": "0.2.1",
+  "version": "0.2.2",
   "unity": "2019.4",
   "license": "MIT",
   "author": "XRTK Team (https://github.com/XRTK)",
   "dependencies": {
-    "com.xrtk.core": "0.2.1"
+    "com.xrtk.core": "0.2.9"
```

Comment:
`xrtk.core` version could be updated to `0.2.11`
We should likely add your package dependency here as well.

Reply:
resolved